packages/components/tsconfig.json

```diff
@@ -60,7 +60,6 @@
 		"src/notice",
 		"src/palette-edit",
 		"src/panel",
-		"src/toolbar",
```

Comment:
Nitty, but should we add the toolbar subfolders that are not typed yet here, instead of `@ts-ignore` at the top of the files?

Reply:
If they're not going to be typed in the next week or so, I think I prefer the `ts-nocheck`s so we can clear out this exclude list and ship the package types 🙈

Reply:
Fwiw, I am starting work on `toolbar-button` today with a general plan to run through the remaining subfolders over the next month or so! 💻